### PR TITLE
add throwing star

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
throwing star that embeds and does 18 damage total

not added to uplink or anything, for future use with ninja

**Media**

https://github.com/space-wizards/space-station-14/assets/39013340/a1020b04-8b39-4152-a95c-974b75130ecb

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
no cl no fun

### CODE REVIEW

File: Resources/Audio/Weapons/attributions.yml [hunk 1]
```diff
@@ -19,3 +19,7 @@
   copyright: "User Mystovski on freesound.org. Modified by LankLTE on github."
   source: "https://freesound.org/people/Mystovski/sounds/201111/"
 
+- files: ["star_hit.ogg"]
+  license: "CC0-1.0"
+  copyright: "Based on genhit3.ogg, modified by deltanedas on github."
```
Comment (deltanedas) [inline Resources/Audio/Weapons/attributions.yml:24]: is this correct thing to do

File: Content.Shared/Projectiles/EmbeddableProjectileComponent.cs [hunk 2]
```diff
@@ -33,4 +34,10 @@ public sealed partial class EmbeddableProjectileComponent : Component
     /// </summary>
     [ViewVariables(VVAccess.ReadWrite), DataField("offset"), AutoNetworkedField]
     public Vector2 Offset = Vector2.Zero;
+
+    /// <summary>
+    /// Sound to play after embedding into a hit target.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite), DataField("sound"), AutoNetworkedField]
+    public SoundSpecifier? Sound;
```
Comment (metalgearsloth) [inline Content.Shared/Projectiles/EmbeddableProjectileComponent.cs:42]: Can this just use the DamageOtherOnHit sound or is it different?

File: Resources/Audio/Weapons/attributions.yml (hunk 1 above)
Comment (metalgearsloth) [reply Resources/Audio/Weapons/attributions.yml:24]: sure as long as the original is CC0-1.0

Comment (deltanedas) [reply Resources/Audio/Weapons/attributions.yml:24]: yeah but it doesnt say the license anywhere so what do i do

File: Content.Shared/Projectiles/EmbeddableProjectileComponent.cs (hunk 2 above)
Comment (deltanedas) [reply Content.Shared/Projectiles/EmbeddableProjectileComponent.cs:42]: afaik embed uses different code from that, the sound is played whenever it embeds which may or may not be the same as whenever it hits idk

File: Resources/Audio/Weapons/attributions.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Audio/Weapons/attributions.yml:24]: @metalgearsloth please clarify

**Comment by deltanedas** [comment]
@metalgearsloth

File: Resources/Audio/Weapons/attributions.yml (hunk 1 above)
Comment (metalgearsloth) [reply Resources/Audio/Weapons/attributions.yml:24]: Just use the license for genhit3.ogg which is probably on TG.

**Comment by metalgearsloth** [comment]
pro

**Comment by deltanedas** [comment]
real

**Pull request merged**